R/keypair.R

```diff
@@ -0,0 +1,92 @@
+# Support for Snowflake key-pair authentication for SPCS and the REST API.
+keypair_credentials <- function(
```

Comment:
Consider using `snowflake_` prefixes for any function that only applies to snowflake.